docs/src/routes/docs/_layout.svelte

```diff
@@ -34,6 +34,10 @@
           title: 'Button',
           segment: 'components/button',
         },
+        {
+          title: 'Badge',
```

Comment:
I thought you were ordering the components alphabetically?

Reply:
Ah, right, button goes next. Will fix that